formlibrary/models.py

```diff
@@ -215,6 +215,7 @@ class CustomForm(models.Model):
     create_date = models.DateTimeField(null=True, blank=True)
     edit_date = models.DateTimeField(null=True, blank=True)
     created_by = models.ForeignKey('auth.User', related_name='customforms', null=True, blank=True, on_delete=models.SET_NULL)
+    is_template = models.BooleanField(default=0)
```

Comment:
It is more common to use False instead of 0

Reply:
0 used for other models, also for is_public and default_global fields in this CustomForm model. I didn't want to change structure.

Reply:
Fair enough